compiler/src/iree/compiler/Codegen/Utils/Utils.cpp

```diff
@@ -832,6 +832,15 @@ int64_t getMinElementBitwidth(linalg::LinalgOp linalgOp) {
   return bitwidth;
 };
 
+constexpr StringLiteral kUnrollLoopName = "unroll_loop";
+void setLoopUnrollMarker(Operation *op) {
+  op->setAttr(kUnrollLoopName, UnitAttr::get(op->getContext()));
+}
+
+bool getLoopUnrollMarker(Operation *op) { return op->hasAttr(kUnrollLoopName); }
```

Comment:
`hasLoopUnrollMarker`? Or alternatively, return the attribute in case we want to specify the unroll amount in the future? The clang version has it: https://clang.llvm.org/docs/AttributeReference.html#unroll-x-unroll

Reply:
nice idea, done!